src/transaction/eth.ts

```diff
@@ -11,11 +11,13 @@ import {CreateRecord, Currency, DeployEthErc20, TransactionKMS, TransferCustomEr
 
 /**
  * Estimate Gas price for the transaction.
- * @param client
  */
-export const ethGetGasPriceInWei = async (client: Web3) => {
-    const {data} = await axios.get('https://ethgasstation.info/json/ethgasAPI.json');
-    return client.utils.toWei(new BigNumber(data.fast).dividedBy(10).toString(), 'gwei');
+export const ethGetGasPriceInWei = async () => {
+  const { data } = await Promise.race([
+    axios.get('https://ethgasstation.info/json/ethgasAPI.json?apiKey=494f9c3a5b3a85d8651e14100558bb6db57ffa5f1120f28dfd1f4f6c7c2a'),
```

Comment:
do not use API Key. This is client lib, 5 req on the customer side should be OK. Alternativelly, create process.env variable so devs could set it correctly